src/app/organizations/organizations/organizations.component.html

```diff
@@ -45,7 +45,7 @@
         <mat-form-field appearance="outline" class="search-form-field">
           <mat-label>Search Organizations</mat-label>
           <input #search matInput formControlName="name" />
-          <button mat-button *ngIf="search.value" matSuffix mat-icon-button (click)="clearSearch()">
+          <button mat-button *ngIf="search.value" matSuffix mat-icon-button (click)="clearSearch()" type="button">
```

Comment:
Nice find! I did not know you had to do that -- the default button type is submit, I just found out. :)